tests/test_simple.py

```diff
@@ -143,6 +143,7 @@ def unbatch(self, output):
         return list(output)
 
 
+@pytest.mark.flaky(retries=5)
```

Comment:
I would be careful with adding retry for `timeout` tests...

Reply:
updated. nice catch. this only applied to batching timeout.